pydantic_ai_slim/pydantic_ai/result.py

```diff
@@ -533,6 +539,18 @@ def timestamp(self) -> datetime:
         else:
             raise ValueError('No stream response or run result provided')  # pragma: no cover
 
+    @property
+    def run_id(self) -> str:
+        """The unique identifier for the agent run."""
+        if self._run_result is not None:
+            assert self._run_result.run_id is not None
+            return self._run_result.run_id
+        elif self._stream_response is not None:
+            assert self._stream_response.run_id is not None
```

Comment:
Same question here, I think we'd always have a `run_id` (or at least we should!)

Reply:
You're right, wasn't needed here. Removed